azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/reducer/ParticipantsReducer.kt

```diff
@@ -4,12 +4,18 @@
 package com.azure.android.communication.ui.chat.redux.reducer
 
 import com.azure.android.communication.ui.chat.redux.action.Action
+import com.azure.android.communication.ui.chat.redux.action.ParticipantAction
 import com.azure.android.communication.ui.chat.redux.state.ParticipantsState
 
 internal interface ParticipantsReducer : Reducer<ParticipantsState>
 
 internal class ParticipantsReducerImpl : ParticipantsReducer {
     override fun reduce(state: ParticipantsState, action: Action): ParticipantsState {
-        return state
+        return when (action) {
+            is ParticipantAction.ParticipantsAdded -> {
```

Comment:
ParticipantsRemoved missing

Reply:
https://github.com/Azure/communication-ui-library-android/pull/481/commits/a909a4b782a91cc6cfa438cbcf3043709f11c75d : Added ParticipantsRemoved to reducer, updated ParticipantsAdded to filter out existing.